src/database/Connector.java

```diff
@@ -0,0 +1,192 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.commons.dbcp2.BasicDataSource;
+
+/**
+ * JDBC connector is needed to connects java program to the database.
+ * Description:
+ * this is the database connector you will need to use to communicate with
+ * the databased hosted on a remote machine.
+ */
+
+public class Connector {
+
+  private BasicDataSource ds;
+  private static String tableName = "UserAccount";
+  private static final String dbName = "CodeU_2017DB";
+  private static final String hostname = "ec2-176-34-225-252.eu-west-1.compute.amazonaws.com";
+  private static final String DBusername = "group34";
+  private static final String DBpassword = "codeu2017";
+
+  public Connector() {
+
+    ds = new BasicDataSource();
+    ds.setDriverClassName("com.mysql.jdbc.Driver");
+    ds.setUsername(DBusername);
+    ds.setPassword(DBpassword);
+    ds.setUrl("jdbc:mysql://" + hostname + ":3306/" + dbName);
+
+  }
+
+  /**
+   * print all the current useNnames
+   *
+   */
+  public List<String> getAllUsers() {
+
+    String sqlSelectUsername = "select username from " + tableName;
+    LinkedList<String> userNames = new LinkedList<>();
```

Comment:
`List<String> userNames = newLinkedList<>();`
Effective Java item 52, refer to objects by their interfaces.